.github/workflows/configs-go/config-release.yml

```diff
@@ -12,3 +12,7 @@ goos: linux
 goarch: amd64
 dir: internal/builders/go/
 binary: slsa-builder-go-{{ .Os }}-{{ .Arch }}
+# ldflags: -s -w is used for reducing the size of the binary.
+ldflags:
+  - "-s"
+  - "-w"
```

Comment:
We're tryying to add a OEL for each file. Can you add it?

Reply:
I can. Isn't this a nit?

Reply:
yep. I also get caught with these myself... I think there's a plugin we can install in VSCode to do that automatically... Prettier?

Reply:
I am fine with nit. I took care of it. We want to be cognizant with new contributions :)

Reply:
Agreed. We don't have a `CONTIBUTION.md`, I think you're in the best position to help us create one and list all the quirks / recommendations /etc you've encountered so far.